packages/playwright-ct-react/index.d.ts

```diff
@@ -17,11 +17,17 @@
 import type {
   TestType,
   PlaywrightTestArgs,
+  PlaywrightTestConfig as BasePlaywrightTestConfig,
   PlaywrightTestOptions,
   PlaywrightWorkerArgs,
   PlaywrightWorkerOptions,
   Locator,
 } from '@playwright/test';
+import type { InlineConfig } from 'vite';
+
+export type PlaywrightTestConfig = Omit<BasePlaywrightTestConfig, 'use'> & {
```

Comment:
Shouldn't you do `export * from '@playwright/test'` to make sure you wrap things properly?